java/adapter/avro/pom.xml

```diff
@@ -25,6 +25,10 @@
   <description>(Contrib/Experimental) A library for converting Avro data to Arrow data.</description>
   <url>http://maven.apache.org</url>
 
+  <properties>
+    <project.build.outputTimestamp>1980-01-01T00:00:02Z</project.build.outputTimestamp>
```

Comment:
Does this force the output timestamp for all builds regardless of build options?

Reply:
Hey @pitrou . The `project.build.outputTimestamp` property is used by several Maven plugins to ensure the consistency of the generated artifacts (assembly, jar, source, ...). They "hook" in their goals on phases where it's required. So, yes, for instance, using `package` phase will set the output timestamp on the artifacts. Does it answer your question (not sure I understood your question correctly ;) ) ?

Reply:
My question is, is this only enabled for specific build variants or options? Or would I get those fixed timestamps if I do a local Java build?

Reply:
It applies to any build including your local one. So you can have a reproducible build locally.

Reply:
Wouldn't it be confusing in some situations then, to have files dated to some random past value?

Reply:
It's only the archives/artifacts. And it's good practice generally speaking (we do that in most of Apache projects).

Reply:
Fair enough. I was expecting this to be enabled selectively when building public packages.